pkg/cluster/reconcile.go

```diff
@@ -272,8 +272,8 @@ func RequestBackupNow(httpClient *http.Client, addr string) error {
 	return nil
 }
 
-func checkBackupExist(httpClient *http.Client, addr string) (bool, error) {
-	resp, err := httpClient.Head(fmt.Sprintf("http://%s/backup", addr))
+func checkBackupExist(httpClient *http.Client, addr, ver string) (bool, error) {
+	resp, err := httpClient.Head(fmt.Sprintf("http://%s/backup?version=%s", addr, ver))
```

Comment:
?etcdVersion

etcdVersion can be different from backup version.

etcd 3.1 can read both 3.0 backup and 3.1 backup.

etcd 3.2 can read both 3.1 backup and 3.2 backup.